flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/utils/WindowUtil.scala

```diff
@@ -217,6 +217,10 @@ object WindowUtil {
           null
         }
         val interval = getOperandAsLong(windowCall.operands(2))
+        if (interval <= 0) {
```

Comment:
otherwise will fail at runtime at https://github.com/apache/flink/blame/cd0cfe42c15042a3c27a0c341416aa32c0d594b6/flink-table/flink-table-runtime/src/main/java/org/apache/flink/table/runtime/operators/window/groupwindow/assigners/TumblingWindowAssigner.java#L50-L53